server/sonar-server/src/main/java/org/sonar/server/organization/ws/UpdateAction.java

```
@@ -68,7 +68,7 @@ public void define(WebService.NewController context) {
 
   @Override
   public void handle(Request request, Response response) throws Exception {
-    userSession.checkPermission(GlobalPermissions.SYSTEM_ADMIN);
+    userSession.checkLoggedIn();
```

fail fast, no need to check DB to then fail because the guy is not logged in